src/app/components/user/register.jsx

```diff
@@ -0,0 +1,93 @@
+import React, { Component } from 'react';
+import { browserHistory } from 'react-router';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import { registerUser } from '../../actions/firebase_actions';
+
+class UserRegister extends Component {
+    constructor(props) {
+        super(props);
+        this.onFormSubmit = this.onFormSubmit.bind(this);
+        this.state = {
+            message: '',
+        };
+    }
+
+    onFormSubmit(event) {
+        event.preventDefault();
+
+        const email = this.refs.email.value;
+        const password = this.refs.password.value;
+        this.props.registerUser({ email, password }).then((data) => {
+            if (data.payload.errorCode) {
+                this.setState({ message: data.payload.errorMessage })
+              ;
+            } else {
+                browserHistory.push('/profile');
+            }
+        }
+    );
+    }
+
+    render() {
```

Comment:
We can remove email, facebook, twitter and github integration so that we only have google login left.